src/test/resources/com/puppycrawl/tools/checkstyle/checks/trailingcomment/InputTrailingComment.java

```diff
@@ -30,4 +30,10 @@ void method2(long ms /* we should ignore this */) {
     final static public String NAME2="Some Name"; /*NOI18N*/
     String NAME3="Some Name"; /*NOI18N
 */
+
+
+    /* A multiline comment here. Should ignore this
+    end here*/                  // don't use trailing comment :)
+
+    /* first comment is ok **/ private int  b; /* Second comment is a trailing comment **/
```

Comment:
Please use `violation` comment in places where violation occurrs

Reply:
fixed